server/index.js

```diff
@@ -34,22 +44,104 @@ app.use([routes.API.DIALCODE.SEARCH, routes.API.ASSET.CREATE], proxy(BASE_URL, {
   })
 );
 
-app.use([ 
-   routes.API.CONTENT.HIERARCHY,
-   routes.API.ASSESSMENT, 
-   routes.API.CONTENT.CREATE,
-   routes.API.CONTENT.BUNDLE,
-   routes.API.CONTENT.UNLISTED_PUBLISH,
-   routes.API.CONTENT.REVIEW_COMMENTS,
-  ], proxy(BASE_URL, {
-    https: true,
-    proxyReqPathResolver: function (req) {
-      return urlHelper.parse(req.originalUrl).path;
-    },
-    proxyReqOptDecorator: proxyUtils.decoratePublicRequestHeaders()
-  })
+/**
+ * @param  {} routes.API.TELEMMETRY This is the telemetry api url
+ * @param  {} {console.log(JSON.stringify(req.body) This is the logging of the request data
+ * @param  {} ;res.status(200) Sending the response 
+ */
+app.post(routes.API.TELEMMETRY, function (req, res) {
+  console.log(JSON.stringify(req.body), 'telemetry logged');
+  res.status(200).json({ message: "telemetry logged" })
+});
+
+/**
+ * @param  {} routes.API.USER_SEARCH This is the user search api url
+ * @param  {} res
```

Comment:
These functional comments are not proper. Make sure you have updated these properly.